biome.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.4.1/schema.json",
+	"organizeImports": {
+		"enabled": false
+	},
+	"files": {
+		"maxSize": 100000000,
+		"ignore": [
+			"dist/*",
```

Comment:
rather than doing this you could have run it once on all files and then you could have configured biome to run only on source dir? and other required directories?

Reply:
makes sense!

Reply:
being monorepo its hard to whitelist files rather than ignore right? no?

Reply:
true